src/axom/inlet/Inlet.cpp

```diff
@@ -189,5 +189,31 @@ void Inlet::writeDoc() {
   }
 }
 
+bool Inlet::verify() {
+  return verifyRecursive(m_sidreRootGroup);
+}
+
+bool Inlet::verifyRecursive(axom::sidre::Group* sidreGroup) {
+  SLIC_ASSERT_MSG(sidreGroup, "Root was nullptr");
+
+  axom::sidre::IndexType viewIndex = sidreGroup->getFirstValidViewIndex();
+  while (axom::sidre::indexIsValid(viewIndex)) {
+    if (sidreGroup->getView(viewIndex)->getName() == "required"
+        && !sidreGroup->getView(viewIndex)->isScalar()) {
+      return false;
```

Comment:
Two small improvements:

* keep a boolean in Inlet whether this failed and start it at the verify() level then return that value.  This way we can verify the whole tree, instead of stopping at the first one.
* print a warning what failed each time with something like:
   
```
 std::string msg = fmt::format("Inlet: {0}: Required field was not specified in Input Deck", fullSidrePathName) // there is a couple ways to get this name
 SLIC_WARNING(msg);
```